tips/rm.txt

```diff
@@ -1,2 +1,6 @@
 i:Use rm -i to be prompted for confirmation before attempting to remove each file
 ri:Use rm -ri to be prompted for confirmation before each directory's contents are processed as well as before the attempt is made to remove the directory
+v:Use rm -v (for verbose) to show the name of the file as it is removed
+d:Use rm -d to attempt to remove directories as well as other types of files
+f:Use rm -f (for force) to attempt to remove files without prompting for confirmation
+rf:**CAUTION** use rm -rf (for recursive force) to force-remove ALL files and folders inside the specified location without any prompt. Do NOT use with / 
```

Comment:
Same as above. Mention regardless of the file's permissions because even rm -r is capable of removing all files and folders as long as the file permissions allow it.